pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -962,60 +963,56 @@ public static void addKitRequests(@NonNull String instanceId, @NonNull String ki
      *      4. BSP and Mercury dummy kit routes in non-prod
      * @return dsm_kit_request_id of the new kit
      * */
-    public static String writeRequest(@NonNull String instanceId, @NonNull String ddpKitRequestId, int kitTypeId,
+    public static String writeRequest(Connection conn, @NonNull String instanceId, @NonNull String ddpKitRequestId, int kitTypeId,
```

Comment:
Added a few more params here so we can capture more information about the kit as part of the same transaction that writes the kit request.